### DISCUSSION TITLE
v2ray c语言版客户端求助

### DISCUSSION THREAD

**Discussion by f1460330800** [v2fly/v2ray-core#1031] [General]
各位大佬：
  最近习v2ray源码，目前v2ray是否支持tun2socks，v2ray是否可以劫持所有本机流量，包括ssh，icmp等协议，目前测试时是无法支持ssh协议的，（全局代理下仍不可通过xshell连接被墙的ip）v2ray 有c语言版客户端吗

**Comment by okudayukiko** [comment]
XShell也要設定Proxy。
不能proxy ICMP。

**Comment by rurirei** [comment]
...?
